internal/host/common.go

```diff
@@ -36,6 +36,7 @@ const (
 	statusInfoInstallationInProgressWritingImageToDiskTimedOut = "Host failed to install because its installation stage $STAGE did not sufficiently progress in the last $MAX_TIME."
 	statusInfoHostReadyToBeMoved                               = "Host is part of pool and is ready to be moved"
 	statusInfoBinding                                          = "Host is waiting tobe bound to the cluster"
```

Comment:
any chance you're fixing it to "to be"?
sorry about that not being related to your PR